components/shared_code/src/shared_data_model/sources/trivy.py

```diff
@@ -0,0 +1,40 @@
+"""Trivy JSON source."""
+
+from pydantic import HttpUrl
+
+from shared_data_model.meta.entity import Color, Entity, EntityAttribute
+from shared_data_model.meta.source import Source
+from shared_data_model.parameters import Severities, access_parameters
+
+TRIVY_JSON = Source(
+    name="Trivy JSON",
+    description="A Trivy vulnerability report in JSON format.",
+    url=HttpUrl("https://aquasecurity.github.io/trivy/v0.45/docs/configuration/reporting/#json"),
```

Comment:
Is there also a non-versioned docs url?

Reply:
I tried `latest` instead of `v0.45` and that also works, so yes!